src/pages/Products/Pro.tsx

```diff
@@ -0,0 +1,226 @@
+                                        })}>
+                                        {item.icon} {item.title}{" "}
+                                        {item.external && (
+                                            <OcLinkexternal2 size={18} />
+                                        )}
+                                    </a>
+                                </h4>
+                                <p>{item.description}</p>
+                            </div>
+                        )}
+                    </For>
+                </div>
+                <div class="target-audience-section slideUp">
+                    <div class="target-header">
+                        <h3>{t("boltz_pro_target_audience_title")}</h3>
+                        <p>{t("boltz_pro_target_audience_subtitle")}</p>
+                    </div>
+                    <div class="target-cards">
+                        <div class="card perfect-for">
+                            <h4>{t("boltz_pro_perfect_for_title")}</h4>
+                            <ul>
+                                <For
+                                    each={[
+                                        { text: t("boltz_pro_perfect_for_1") },
+                                        { text: t("boltz_pro_perfect_for_2") },
+                                        { text: t("boltz_pro_perfect_for_3") },
+                                    ]}>
+                                    {(item) => (
+                                        <li>
+                                            <VsCheck />
+                                            {item.text}
+                                        </li>
+                                    )}
+                                </For>
+                            </ul>
+                        </div>
+                        <div class="card not-designed-for">
+                            <h4>{t("boltz_pro_not_designed_for_title")}</h4>
+                            <ul>
+                                <For
+                                    each={[
+                                        {
+                                            text: t(
+                                                "boltz_pro_not_designed_for_1",
+                                            ),
+                                        },
+                                        {
+                                            text: t(
+                                                "boltz_pro_not_designed_for_2",
+                                            ),
+                                        },
+                                        {
+                                            text: t(
+                                                "boltz_pro_not_designed_for_3",
+                                            ),
+                                        },
+                                    ]}>
```

Comment:
```suggestion
                                    each={[
                                        t("boltz_pro_not_designed_for_1"),
                                        t("boltz_pro_not_designed_for_2"),
                                        t("boltz_pro_not_designed_for_3")                                      },
                                    ]}>
```
This feels a bit bloated